ccd-definition/ComplexTypes/DQ/RequestedCourt.json

```diff
@@ -16,12 +23,21 @@
   },
   {
     "ID": "RequestedCourt",
-    "ListElementCode": "name",
-    "FieldType": "Text",
-    "ElementLabel": "Court name",
+    "ListElementCode": "courtFinderLabel",
+    "FieldType": "Label",
+    "ElementLabel": "<a target=\"_blank\" href=\"courttribunalfinder.service.gov.uk/search/address\">Use court finder</a> (opens in a new window) to find the 3 digit court code of your preferred court.",
```

Comment:
You're missing https:// at the front of the courttribunalfinder url

Reply:
Hi Doug, this has been added now👍🏽